### DISCUSSION TITLE
feature request: Do not show skipped jobs in Pull Request Checks

### DISCUSSION THREAD

**Discussion by supershal** [community/community#44142] [Actions]
### Select Topic Area

Product Feedback

### Body

We have several jobs that runs on PR label only.  However when PR label is not created, those jobs still shows up as "skipped" "grayed out" in the Pull request Status check panel. This makes it difficult to filter through jobs that actually runs on a PR.

Can we either 
Do not show skipped jobs in Pull request check status panel.
Or
Sort the list of jobs in status panel such that "required" jobs shows first followed by "optional" and "skipped" at last.

**Comment by triple3567** [comment]
bump. This feature is also requested in another [discussion](https://github.com/orgs/community/discussions/18001#discussion-4123764)

**Comment by michaelschuett-tomtom** [comment]
yeah this would be cool....

**Comment by thomas-bc** [comment]
Bumping this
